tests/robot-tests/tests/libs/admin-common.robot

```diff
@@ -310,3 +310,185 @@ user enters text into meta guidance data file content editor
+    user selects theme and topic from admin dashboard  %{TEST_THEME_NAME}  %{TEST_TOPIC_NAME}
+    user waits until page contains accordion section   ${PUBLICATION_NAME}
+    user opens accordion section  ${PUBLICATION_NAME}
+    ${accordion}=  user gets accordion section content element  ${PUBLICATION_NAME}
+    user opens details dropdown   ${RELEASE_NAME} ${RELEASE_STATUS}  ${accordion}
+    ${details}=  user gets details content element  ${RELEASE_NAME} ${RELEASE_STATUS}  ${accordion}
+    user waits until parent contains element   ${details}   xpath:.//a[text()="View this release"]
+    user clicks button  Amend this release  ${details}
+    user clicks button  Confirm
+
+user replaces data files for amendment 
+    [Arguments]  ${SUBJECT_NAME}  ${SUBJECT_FILE}  ${META_FILE}  ${text}  ${ERROR}
+    [Documentation]  EES-1442
+    user waits until page contains element  id:dataFileUploadForm-subjectTitle
+    user waits until h2 is visible  Uploaded data files
+    user waits until page contains accordion section   ${SUBJECT_NAME}
+    user opens accordion section   ${SUBJECT_NAME}
+    ${section}=  user gets accordion section content element  ${SUBJECT_NAME}
+    user clicks link  Replace data  ${section}
+    user chooses file   id:dataFileUploadForm-dataFile       ${FILES_DIR}${SUBJECT_FILE}
+    user chooses file   id:dataFileUploadForm-metadataFile   ${FILES_DIR}${META_FILE}
+    user clicks button  Upload data files
+    user clicks button  Confirm
+    #EES-1442: Bug when Confirm data replacement button doesn't show
+    user reloads page
+    user waits until page contains  Footnotes: OK
+    user waits until page contains  Data blocks: OK
+    user waits until button is enabled  Confirm data replacement
+
+user deletes subject file 
+    [Arguments]  ${SUBJECT_NAME}
+    user waits until page contains accordion section  ${SUBJECT_NAME}
+    user opens accordion section  ${SUBJECT_NAME}
+    user scrolls to accordion section content  ${SUBJECT_NAME}
+    ${accordion}=  user gets accordion section content element  ${SUBJECT_NAME}
+    ${button}=  user gets button element  Delete files  ${accordion}
+    user clicks element  ${button}
+    user clicks button  Confirm
+
+user approves release for immediate publication
+    user clicks link  Sign off
+    user clicks button  Edit release status
+    user waits until h2 is visible  Edit release status
+    user clicks radio   Approved for publication
+    user enters text into element  id:releaseStatusForm-internalReleaseNote  Approved by UI tests
+    user clicks radio   As soon as possible
+    user clicks button   Update status
+    user waits until h2 is visible  Sign off
+    user checks summary list contains  Current status  Approved
+    user waits for release process status to be  Complete    ${release_complete_wait}
+    user reloads page  # EES-1448
+    user checks page does not contain button  Edit release status
+
+user navigates to admin dashboard 
```

Comment:
I'd merge this keyword with `user navigates to admin dashboard as analyst1` and accept the username as an argument, so you can do `user navigates to admin dashboard  Bau1`